src/proj7BittingCerratoCohenEllmer/bantam/lexer/Scanner.java

```diff
@@ -186,26 +152,34 @@ private boolean isCompleteComment(Stack<Character> spellingStack) {
         char secondChar = spellingStack.get(1);
         char lastChar = spellingStack.peek();
         if (secondChar == '/') {
-            return lastChar == '\n' || lastChar == '\r';
+            if (lastChar == '\n' || lastChar == '\r') {
+                skippedLastToken = '\0';
```

Comment:
Should this be popped from the comment before creating the token (dont include newlines in single line comment)?